doctor.py

```diff
@@ -346,6 +347,43 @@ def check_duplicate_5_tuples(self):
             
         return not bool(bad)
 
+    def check_connection_distribution(self):
+        """Checking if connections are unevenly distributed across workers
+
+        Usually, connections should be distributed evenly across workers. If connections are
+        unevenly distributed, load balancing might be not working properly.
+        """
+
+        files = find_recent_log_files(self.log_directory, "conn.*", days=1)
+        if not files:
+            self.err("No conn log files in the past day???")
+            return False
+
+        nodes = defaultdict(int)
+        for rec in read_bro_logs_with_line_limit(reversed(files), 10000):
+            # Only count connections that have completed a three way handshake
+            # Also ignore flipped connections as those are probably backscatter
+            if 'h' not in rec['history'].lower() or '^' in rec['history']:
+                continue
+            # Also ignore connections that didn't send bytes back and forth
+            if rec.get('orig_bytes') == '0' or rec.get('resp_bytes') == '0':
+                continue
+            node = rec.get('_node_name', 'bro')
+            nodes[node] += 1
+
+        mean = float(sum(nodes.values())) / len(nodes)
+        variance = reduce(lambda var, cnt: var + (cnt - mean)**2, nodes.values(), 0) / len(nodes)
+        rsd = sqrt(variance) / mean
+
+        if rsd > 0.1:
```

Comment:
Should we have a 

    if len(nodes) == 1:
        self.ok("Only one worker appears to be in use, unable to check distribution")